generators/utils.js

```diff
@@ -414,11 +414,13 @@ function decodeBase64(string, encoding = 'utf-8') {
  * Get all the generator configuration from the .yo-rc.json file
  * @param {Generator} generator the generator instance to use
  * @param {boolean} force force getting direct from file
+ * @param {String} base path where the .yo-rc.json file is located. Default is cwd.
  */
-function getAllJhipsterConfig(generator, force) {
+function getAllJhipsterConfig(generator, force, basePath = '') {
```

Comment:
Can you check if this is exposed anywhere binnour public API if so we need to take care of backwards compatibility

Reply:
Isn't that being taken care of by the default parameter?

Reply:
So I just added unit tests as well and an extra check within `utils.getAllJhipsterConfig()`. 

The method is not used directly within the generator but rather sub-gens use the one from `generator-base.js`. Nevertheless, since the method is exported in `utils.js` it could be used by 3rd party code(e.g. blueprints,modules,etc.)

Reply:
sorry didnt notice default param